toolbox

```diff
@@ -1040,9 +1041,11 @@ list_images()
         return 1
     fi
 
-    if ! output=$(echo "$details" \
-                  | sed "s/ \{2,\}/\t/g" 2>&3 \
-                  | column --separator "$tab" --table --table-columns "IMAGE ID,IMAGE NAME,CREATED" 2>&3); then
+    table_data=$(printf "%s\t%s\t%s\n" "IMAGE ID" "IMAGE NAME" "CREATED"; \
+                 echo "$details" \
+                 | sed "s/ \{2,\}/\t/g" 2>&3)
+    if ! output=$(echo "$table_data" \
+                  | column -s "$tab" -t 2>&3); then
```

Comment:
Does `bsdmainutils` not have the longer form of the options? ie., `--separator`, `--table`.

Reply:
Unfortunately it doesnt - see https://manpages.debian.org/buster/bsdmainutils/column.1.en.html.

Reply:
Ok, fine. Thanks for the link.